WordPress/Classes/Extensions/UIColor+MurielColors.swift

```diff
@@ -0,0 +1,150 @@
+    case hotOrange = "Hot-Orange"
+    case hotPink = "Hot-Pink"
+    case hotPurple = "Hot-Purple"
+    case hotRed = "Hot-Red"
+    case hotYellow = "Hot-Yellow"
+    case jetpackGreen = "Jetpack-Green"
+    case orange
+    case pink
+    case purple
+    case red
+    case wooPurple = "Woo-Purple"
+    case yellow
+
+    var description: String {
+        return rawValue.capitalized
+    }
+}
+
+/// Value of a Muriel color's shade
+///
+/// Note: There are a finite number of acceptable values. Not just any Int works.
+///       Also, enum cases cannot begin with a number, thus the `shade` prefix.
+enum MurielColorShade: Int, CustomStringConvertible {
+    case shade0 = 0
+    case shade50 = 50
+    case shade100 = 100
+    case shade200 = 200
+    case shade300 = 300
+    case shade400 = 400
+    case shade500 = 500
+    case shade600 = 600
+    case shade700 = 700
+    case shade800 = 800
+    case shade900 = 900
+
+    var description: String {
+        return "\(rawValue)"
+    }
+}
+
+struct MurielColorIdentifier {
+    let name: MurielColorName
+    let shade: MurielColorShade
+
+    init(name: MurielColorName, shade: MurielColorShade = .shade500) {
+        self.name = name
+        self.shade = shade
+    }
+
+    init(from identifier: MurielColorIdentifier, shade: MurielColorShade) {
+        self.name = identifier.name
+        self.shade = shade
+    }
+
+    // MARK: - Muriel's semantic colors
+    static let accent = MurielColorIdentifier(name: .hotPink)
+    static let divider = MurielColorIdentifier(name: .gray, shade: .shade50)
+    static let error = MurielColorIdentifier(name: .hotRed)
+    static let neutral = MurielColorIdentifier(name: .gray)
+    static let primary = MurielColorIdentifier(name: .blue)
+    static let success = MurielColorIdentifier(name: .green)
+    static let text = MurielColorIdentifier(name: .gray, shade: .shade800)
+    static let textSubtle = MurielColorIdentifier(name: .gray, shade: .shade500)
+    static let warning = MurielColorIdentifier(name: .hotYellow)
+
+    // MARK: - Additional iOS semantic colors
+    static let navigationBar = MurielColorIdentifier(name: .blue)
+    static let navigationBarShadow = MurielColorIdentifier(name: .blue, shade: .shade800)
+
+    /// The full name of the color, with required shade value
+    func assetName() -> String {
+        return "\(name)-\(shade)"
+    }
+}
+
+extension UIColor {
+    /// Get a UIColor from the Muriel color palette
+    ///
+    /// - Parameters:
+    ///   - color: an instance of a MurielColorIdentifier
+    ///   - shade: an optional shade value
+    /// - Returns: UIColor. Red in cases of error
+    class func muriel(color colorID: MurielColorIdentifier) -> UIColor {
+        let assetName = colorID.assetName()
+        guard let color = UIColor(named: assetName) else {
+            return .red
+        }
+        return color
+    }
+
+    /// Muriel accent color
+    static var accent = muriel(color: .accent)
```

Comment:
I feel like perhaps these var names are all a bit general to sit directly in UIColor without any kind of name spacing. The easiest solution might just be to prepend `muriel` to them all, although the duplication is a bit annoying. 

Other ideas: 

* Put them all in a `Muriel` struct within `UIColor`, but then you'll have to explicitly reference `UIColor.` at any call sites.
* Add a `color` var to `MurielColorIdentifier` to return the color directly from there. Again you'll have duplication when using it though, as we'd have to do `MurielColorIdentifier(name: .accent).color`.

Just some ideas! I may also just be nitpicking!